Readme.md

```diff
@@ -493,3 +500,12 @@ Todo
 - make routing a mw step, to help w/ path rewriting (to route, edit, re-route)
 - support limit on max request size? (error out if too big)
 - call versioning
+
+
+[express]: https://expressjs.com/
```

Comment:
I like the links to the websites, but I don't like losing the links to the npmjs packages.  Ideas on how to do both?  (However, see note above about now having lost the [restiq] npmjs link.)